qhub/template/stages/04-kubernetes-ingress/modules/kubernetes/ingress/main.tf

```diff
@@ -1,3 +1,23 @@
+locals {
+  default_cert = [
```

Comment:
Don't you wish terraform had more expressive conditional operators...

Reply:
I would suggest something like. Idea taken from https://stackoverflow.com/questions/68528209/terraform-switch

```terraform
locals {
  default_cert = [
    "--entrypoints.websecure.http.tls.certResolver=default",
    "--entrypoints.minio.http.tls.certResolver=default",
  ]
  certificate-settings = {
    self_signed = local.default_cert
    existing = local.default_cert
    letsencrypt = [
      "--entrypoints.websecure.http.tls.certResolver=letsencrypt",
      "--entrypoints.minio.http.tls.certResolver=letsencrypt",
      "--certificatesresolvers.letsencrypt.acme.tlschallenge",
      "--certificatesresolvers.letsencrypt.acme.email=${var.acme-email}",
      "--certificatesresolvers.letsencrypt.acme.storage=acme.json",
      "--certificatesresolvers.letsencrypt.acme.caserver=${var.acme-server}",
    ]
    disabled = []
  }
  disabled        = "${var.certificate-service}" == "disabled" ? [] : []
  add-certificate = local.certificate-settings[var.certificate-service]
}
```